generate.sh

```diff
@@ -18,12 +20,14 @@ done
 
 echo ""
 echo "*** Build With Jekyll ***"
-export JEKYLL_VERSION=3.8
+export JEKYLL_VERSION=4.2.2
 docker run --rm \
     -e "JEKYLL_UID=$(id -u)" \
     -e "JEKYLL_GID=$(id -g)" \
     -v "$PWD/docs:/srv/jekyll:Z" \
     -w "/srv/jekyll" \
+    --add-host "rubygems.org:151.101.193.227" \
+    --add-host "index.rubygems.org:151.101.193.227" \
```

Comment:
For some strange reason, I had to hard-code these IPs. I didn't used to have to do that, and Jekyll 3.8 no longer seems to work.

Reply:
This turned out to be a problem with my VM networking.